tests/testapp/tests/test_permission_classes.py

```diff
@@ -117,3 +121,29 @@ def test_organization_field_with_errored_parent(self):
         with self.assertRaises(AttributeError) as error:
             self.client.get(reverse('test_error_field_view'), **auth)
         self.assertIn('Organization not found', str(error.exception))
+
+    def test_custom_django_model_permission_with_view_permission(self):
```

Comment:
Can we have a test for negative case when user does not have the permission; they are given 403.

Reply:
I have updated the tests. :+1: